### ISSUE TITLE
ALPX-3 domain simulation in wrong DRS directory

### ISSUE THREAD

**Issue by jesusff** [FPS-URB-RCC/STAGE-0_Analysis#10]
The UCAN simulation in the ALPX-3 domain is under the PARIS-3 folder:

FPS-URB-RCC/**PARIS-3**/UCAN/ERA5/evaluation/r1i1p1f1/WRF451R-ALPX/v1/1hr/tas/v20240703/tas_**ALPX-3**_ERA5_evaluation_r1i1p1f1_UCAN_WRF451R-ALPX_v1_1hr_202003010000-202009010000.nc

**Comment by yoselita** [comment]
@jesusff it is done on purpose, as we do not have permission to create a folder with a new domain name. We cannot create folder FPS-URB-RCC/**ALPX-3**/ as it should be.

**Comment by jesusff** [comment]
We can either ask Peter to create the folder for this domain, or file it under PARIS-3, but then we should match the domain_id in the filename. The source_id will give a hint (WRF451R-**ALPX**) that this simulation is actually run on a larger domain

**Comment by yoselita** [comment]
So we stay in PARIS-3 folder, I should change domain_id to PARIS-3?

**Comment by jesusff** [comment]
yes, this folder and the first part of the name are the same DRS element (domain_id, also inside the file in the metadata). In this case, the domain change can be considered a sensitivity analysis of the domain size of PARIS-3.

The best approach would be to place it in the right folder, but there we also risk not being located for other analyses.

**Comment by yoselita** [comment]
To ovoid the risk, I opt for changing the domain_id then.

**Comment by yoselita** [comment]
The UCAN APLX-3 data updated:
1. domain updated in the filename: ALPX-3 to PARIS-3
2. `domain_id` updated in global attributes from "ALPX-3" to "PARIS-3"
3. `domain` in global attributes updated from "ALP eXtended" to "Paris extended"
